atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/UniqueSchemaMutationLockTableTest.java

```diff
@@ -83,8 +86,10 @@ public void shouldReturnTheLockTableIfItIsTheOnlyOne() throws TException {
     }
 
     @Test
-    public void shouldThrowExceptionIfMultipleTablesExist() throws Exception {
+    public void shouldThrowExceptionIfMultipleTablesNonEmptyExist() throws Exception {
         when(lockTables.getAllLockTables()).thenReturn(ImmutableSet.of(lockTable1, lockTable2));
+        when(lockTables.isLockTableEmpty(lockTable1)).thenReturn(false);
```

Comment:
`lockTable1` and `lockTable2` should actually be `private static final` and therefore `LOCK_TABLE_1/2`.